src/pages/Admin/AdminVenueRoomDetails.tsx

```diff
@@ -44,18 +47,16 @@ export const AdminVenueRoomDetails = ({
 
   const events = useSelector((state) => state.firestore.ordered.events);
 
-  const filteredEvents =
-    events &&
-    events.filter((e) => {
-      if (e.room === room.title) {
-        return e;
-      }
-      return null;
-    });
+  const filteredEvents = useMemo(() => {
+    if (!events) return;
+
+    return events.filter((event) => (event.room === room.title ? event : null));
```

Comment:
looks like for `filter` we just need a boolean result
```suggestion
    return events.filter((event) => event.room === room.title);
```

Reply:
True that! Fixed in https://github.com/sparkletown/sparkle/pull/1775/commits/d26707ce8ee6e68c1abf87d8de885f6572c0edaa